source/_integrations/zha.markdown

```diff
@@ -316,6 +316,20 @@ To add a new device:
 1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
+#### Best practices for avoiding pairing difficulties
```

Comment:
```suggestion
### Best practices for avoiding pairing difficulties
```

titles should be the same or +/- as the previous title

Is it correct that this should be under "Adding devices"?
Is not under troubleshooting a better place?